products/BellHybrid/apps/application-bell-settings/models/alarm_settings/AlarmSettingsListItemProvider.hpp

```diff
@@ -11,10 +11,10 @@ namespace app::bell_settings
     class AlarmSettingsListItemProvider : public SettingsListItemProvider
     {
       public:
-        AlarmSettingsListItemProvider(AbstractAlarmSettingsModel &model, std::vector<UTF8> alarmToneRange);
+        AlarmSettingsListItemProvider(AbstractAlarmSettingsModel &model, std::vector<std::string> alarmToneRange);
```

Comment:
afair I understand this is forced by check for stl type in template now. But utf8 is first class citizen in the os - can't we just add this case to the template instead of changing everywhere `UTF8 -> std::string` ?

Reply:
True, fixed.